gpii/node_modules/testing/src/RunTestDefs.js

```diff
@@ -0,0 +1,240 @@
+/*
+
+    A collection of functions to expand "test defs" and run them as tests.
+
+    Copyright 2019 Raising the Floor International
+
+    Licensed under the New BSD license. You may not use this file except in
+    compliance with this License.
+
+    You may obtain a copy of the License at
+    https://github.com/gpii/universal/LICENSE.txt
+
+*/
+"use strict";
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+var kettle = require("kettle");
+
+var path = require("path");
+
+// TODO: Replace all usages of kettle.test.bootstrapServer
+
+// TODO: Organize and add docs throughout.
+
+/**
+ *
+ *  A function that wraps test definitions in a test environment, case holder, and runs them.  Uses
+ *  sequence elements instead of manipulating arrays.  Intended as a long-term replacement for
```

Comment:
Again eliminate comment referring to deleted definition - these just distract the contemporary reader

Reply:
It was a courtesy to people transitioning from the old world to the new, I'm fine with removing both.